### PR TITLE
Bump Goodstrap and update Header

### PR DESCRIPTION
**Pull request by Pkingo**
- Bump Goodstrap
- expose "disableHelpDropdown" and "onHelpClicked" in Header API

JIRA: CAT-329

---

Supported PR commands:

| Command         | Description            |
| --------------- | ---------------------- |
| `ok to test`    | Re-run standard checks |
| `extended test` | BackstopJS tests       |

---

# PR Checklist

-   [x] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [x] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [x] `check` passes
-   [x] `check-extended` passes
-   [x] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Merge Failed.

Zuul merger could not merge this change into the base branch. This is most likely caused by merge conflicts. Please rebase the change and upload the rebased version. In case of further errors, contact the zuul administrator.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1818/ : SUCCESS in 1m 22s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3427/ : FAILURE in 31s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1819/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3428/ : FAILURE in 3m 31s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1825/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3436/ : SUCCESS in 11m 06s

**Comment by Pkingo** [comment]
extended test

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1832/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/550/ : SUCCESS in 31m 54s

File: examples/sdk-examples/package.json [hunk 1]
```diff
@@ -84,7 +84,7 @@
     "dependencies": {
         "@gooddata/api-client-bear": "^8.2.0-alpha.16",
         "@gooddata/api-model-bear": "^8.2.0-alpha.16",
-        "@gooddata/goodstrap": "^70.0.2",
+        "@gooddata/goodstrap": "~70.1.0",
```
Comment (no23reason) [inline examples/sdk-examples/package.json:87]: Please use carets. There is a `--caret` switch in `rush add`.

File: libs/sdk-ui-kit/src/Header/Header.tsx [hunk 2]
```diff
@@ -75,6 +75,9 @@ export interface IAppHeaderProps {
     activeColor?: string;
 
     userName: string;
+
+    disableHelpDropdown?: boolean;
+    onHelpClicked?: boolean;
```
Comment (no23reason) [inline libs/sdk-ui-kit/src/Header/Header.tsx:80]: Please use present tense: `onHelpClick` as is already used in other props here.

**Review by no23reason — CHANGES_REQUESTED** [review]

File: examples/sdk-examples/package.json (hunk 1 above)
Comment (Pkingo) [reply examples/sdk-examples/package.json:87]: Thanks, I'll change it (:

**Comment by gdgate** [comment]
Merge Failed.

Zuul merger could not merge this change into the base branch. This is most likely caused by merge conflicts. Please rebase the change and upload the rebased version. In case of further errors, contact the zuul administrator.

File: libs/sdk-ui-kit/src/Header/Header.tsx (hunk 2 above)
Comment (Pkingo) [reply libs/sdk-ui-kit/src/Header/Header.tsx]: You got it 😄

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1841/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3450/ : SUCCESS in 11m 09s

**Comment by Pkingo** [comment]
extended test

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1842/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/551/ : SUCCESS in 25m 46s

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/588/ : FAILURE in 7m 26s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/849/ : SUCCESS in 20s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3453/ : SUCCESS in 14m 48s

**Pull request merged**